app/views/provider_interface/interviews/check.html.erb

```diff
@@ -13,7 +13,7 @@
     <% if @interview.present? %>
       <%= govuk_button_to t('.update.confirm'), { action: 'update', id: @interview.id }, method: :put %>
     <% else %>
-      <%= govuk_button_to t('.confirm'), action: 'commit' %>
+      <%= govuk_button_to t('.confirm'), { action: 'commit' }, method: :post %>
```

Comment:
@peteryates This seems surprising to me. Do HTML attributes always need to be in `{ }`, and why not `method`?

Reply:
I think it has to do with the button_to implementation https://apidock.com/rails/ActionView/Helpers/UrlHelper/button_to

Reply:
Ah, `#govuk_button_to` is a pretty much a wrapper for [`#button_to`](https://api.rubyonrails.org/v6.1.0/classes/ActionView/Helpers/UrlHelper.html#method-i-button_to), which takes its `options` argument as the second arg and `html_options` as the third (which here is `method: :post`).

The only thing the govuk variant does is forces in a `govuk-button` class.